transport/http_util.go

```diff
@@ -144,18 +147,37 @@ func isWhitelistedPseudoHeader(hdr string) bool {
 }
 
 func validContentType(t string) bool {
-	e := "application/grpc"
-	if !strings.HasPrefix(t, e) {
+	if !strings.HasPrefix(t, baseContentType) {
 		return false
 	}
 	// Support variations on the content-type
 	// (e.g. "application/grpc+blah", "application/grpc;blah").
-	if len(t) > len(e) && t[len(e)] != '+' && t[len(e)] != ';' {
+	if len(t) > len(baseContentType) && t[len(baseContentType)] != '+' && t[len(baseContentType)] != ';' {
 		return false
 	}
 	return true
 }
 
+func getContentSubtype(t string) string {
```

Comment:
This is just using the same variable name as `validContentType`, which is similar.